src/runtimes/node/bundlers/index.ts

```diff
@@ -86,5 +90,37 @@ const getBundler = (name: NodeBundlerName): NodeBundler => {
   }
 }
 
-export { getBundler }
+// We use ZISI as the default bundler, except for certain extensions, for which
+// esbuild is the only option.
+const getDefaultBundler = async ({
```

Comment:
This function was moved from `src/runtimes/node/index.ts`. The code was not changed.